tools/toolchains/arm.py

```diff
@@ -19,15 +19,16 @@
 
 import re
 from copy import copy
-from os.path import join, dirname, splitext, basename, exists, isfile
-from os import makedirs, write, remove
+from os.path import join, dirname, splitext, basename, exists, isfile, split
+from os import makedirs, write, curdir, remove
```

Comment:
I don't think `curdir` is used. Could you remove this? or point me to where you use it?

Reply:
curdir is removed